phpunit.xml.dist

```diff
@@ -13,6 +18,7 @@
     convertWarningsToExceptions="true"
     processIsolation="false"
     stopOnFailure="false"
+    verbose="true"
```

Comment:
with this, do we still need to add `-v` when we run tool under Travis/AppVeyor/CircleCI ?

Reply:
I don't think so, let me remove it!